blueoil/cmd/train.py

```diff
@@ -217,7 +217,18 @@ def start_training(config):
         progbar.update(last_step)
     for step in range(last_step, max_steps):
 
-        images, labels = train_dataset.feed()
+        samples_dict = train_dataset.feed()
+
+        # TODO: temporary solution, update when supporting network_multi
+        images = samples_dict["image"]
+        if config.TASK == Tasks.OBJECT_DETECTION:
+            labels = samples_dict["gt_boxes"]
+        elif config.TASK == Tasks.SEMANTIC_SEGMENTATION:
+            labels = samples_dict["mask"]
+        elif config.TASK == Tasks.KEYPOINT_DETECTION:
+            labels = samples_dict["heatmap"]
+        else:
+            labels = samples_dict["label"]
```

Comment:
This code is repeated in this file. Can you avoid the duplicated code?

Reply:
@tfujiwar 
Thanks for comments! I have fixed it. But it is still a temporary solution. It will be updated in next PRs.